willa/web/app.py

```diff
@@ -34,6 +37,22 @@
 ]
 
 
+async def get_step(self, step_id: str) -> Optional[StepDict]:
+    # Get step and related feedback
+    query = """
+    SELECT  s.*,
+            f.id feedback_id,
+            f.value feedback_value,
+            f."comment" feedback_comment
+    FROM "Step" s left join "Feedback" f on s.id = f."stepId"
```

Comment:
SQL style nit:
```suggestion
    FROM "Step" s LEFT JOIN "Feedback" f ON s.id = f."stepId"
```

Reply:
Yeah, I wrote it that way based on upstream's style, not ours.  I definitely agree that looks better.